README.rst

```diff
@@ -82,6 +82,21 @@ On a Debian based system, the installation is as follows::
     make && make install
     cd ..
 
+Installing the PostgresSQL session_exec extension
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is optional, but required if you choose to install the python packages
+into a virtualenv.
+
+::
+    git clone https://github.com/okbob/session_exec
```

Comment:
I wasn't able to build session_exec for postgres 9.3 (what we were using on travis) because `session_preload_libraries` is not available for postgres 9.3...  Is it meant to work?